activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Preserve column comment value on changing column name
```

Comment:
This already worked on PostgreSQL, so we should mention MySQL here specifically. Can you add a full stop at the end of the sentence too?

Reply:
Thanks, fixed now.